styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Enterprise Managed Services -->
+   <div class="container-fluid feature-container bg-crumpled-paper">
+
+      <div class="row">
+        <!-- Left Column -->
+        <div class="col-md-5 no-padding">
+            <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/office-365/sidebyside.png">
+        </div>
+
+        <!-- Right Column -->
+        <div class="col-md-7">
+          <div class="col-md-11 half-padding">
+              <h2>Managed Services for Office 365.</h2>
+              <p>Maximize your Office 365 investment with advisory services and Enterprise-class support for Office 365. Get more out of your Office 365 investment.</p>
+          </div>
+          <!-- Feature 1 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb idio-high-five black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Work Better Together</h3>
+              <p>Collaborate easier with access to email, calendar, files and Offfice applications from any device, on the road, at home or at the office.</p>
+            </div>
+          </div>
+          <!-- Feature 2 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb auto-process black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Never Update Again</h3>
+              <p>Always enjoy access to the latest versions, feature releases and security updates of Microsoft Office, Outlook, SharePoint, and more. <a href="javascript:">Office 2016 now available.</a></p>
+            </div>
+          </div>
+          <!-- Feature 3 -->
+          <div class="col-md-12 feature">
+            <div class="col-md-2 col-sm-1">
+              <i class="rsweb gen-locked-checked-shield black"></i>
+            </div>
+            <div class="col-md-10 col-sm-11">
+              <h3>Work More Securely</h3>
+              <p>Work and share files confidently with built-in, customizable security and privacy controls that help keep your critical data safe.</p>
+            </div>
+          </div>
+          <!-- CTA -->
+          <div class="col-md-12 center">
+            <a href="javascript:" class="button">View Plans and Pricing</a>
+          </div>
+        </div>
+      </div>
+      <div class="row bg-medium-teal">
```

Comment:
rows have padding so the background wont span the full width here. If you want it to you should put the background onto a <div class="container-fluid"> which would go around the row.